index.html

```diff
@@ -1,49 +1,51 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
```

Comment:
Please undo the spacing changes to this file, it makes it look like there are more changes than there really are

Reply:
I think this is what's currently merged in main. Any specific branch index.html that's "the one"? :)

Reply:
I tried to add Amruthas changes to the old formatting. The problem is that VS Code normally formats on save. So I needed to save without formatting